tests/testOutputRoot.py

```diff
@@ -85,8 +85,13 @@ def testCreateOutputRoot(self):
         self.assertTrue(os.path.isdir(testOutput))
         self.assertTrue(os.path.exists(os.path.join(testOutput, "repositoryCfg.yaml")))
         cfg = dafPersist.PosixStorage.getRepositoryCfg(testOutput)
+        expectedCfg = dafPersist.RepositoryCfg(root=testPath,
+                                               mapper=MinMapper1,
+                                               mapperArgs=None,
+                                               parents=None,
+                                               policy=None)
         self.assertEqual(len(cfg.parents), 1)
-        self.assertEqual(cfg.parents[0], testPath)
+        self.assertEqual(cfg.parents[0], expectedCfg)
```

Comment:
Nitpick - combine two tests into `self.assertEqual(cfg.parents, [expectedCfg])` (just as it is done below)?